packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -41,7 +41,7 @@ const FilterDropdown = ({
             handleFilterChange(event.target.value);
         }
     };
-    if (isMobile()) {
+    if (isMobile() && label) {
```

Comment:
Previously even if the `label === undefined `the component `<SelectNative />` was rendered and now it won't. Please, make sure that it will be tested

Reply:
removed